File: app/models/post.rb
class Post < ApplicationRecord
  belongs_to :customer
  has_many :likes

  attr_accessor :first_name, :last_name

  def json_format
    customer = Customer.find(customer_id)
    {
      id: id,
      message: message,
      created_at: created_at,
      updated_at: updated_at,
      customer_id: customer_id,
      first_name: customer.first_name,
      last_name: customer.last_name
    }
  end
end

File: Frontend/acebook-frontend/src/components/Post.js
import React from 'react';

const Post = ({post}) =>
  <div className="single-post" key={post.id}>
    <p>{post.message}</p>
    <p>{post.created_at}</p>
    <p>{post.customer_id}</p>
  </div>



  export default Post;
